CMakeLists.txt

```
@@ -31,8 +31,14 @@ if(NOT DEFINED CMAKE_INSTALL_PREFIX OR CMAKE_INSTALL_PREFIX STREQUAL "")
 endif()
 message(STATUS "CMAKE_INSTALL_PREFIX = ${CMAKE_INSTALL_PREFIX}")
 
+set(SVT_VERSION_MAJOR           "1")
+set(SVT_VERSION_MINOR           "4")
+set(SVT_VERSION_PATCHLEVEL      "0")
```

```suggestion
set(SVT_HEVC_VERSION_PATCHLEVEL      "0")
```

Change to SVT_HEVC to separate out from the other SVT encoders?

---

Does that mean only cmake variables ? If that is right, I applied it :)

---

Yes